### PR TITLE
feat: support contact -> account associations in engagement vertical

### PR DESCRIPTION
**Pull request by asdfryan**
Tested locally

### CODE REVIEW

File: packages/core/destination_writers/bigquery.ts [hunk 1]
```diff
@@ -1340,28 +1345,8 @@ const schemaByCommonObjectType: {
...
-        },
-        {
-          name: 'open_count',
-          type: 'INTEGER',
-          mode: 'REQUIRED',
-        },
-        {
-          name: 'opt_out_count',
-          type: 'INTEGER',
+          name: 'metrics',
```
Comment (asdfryan) [inline packages/core/destination_writers/bigquery.ts:1348]: This is an unrelated fix -- bigquery was outdated for a while.

**Review by lucasmarshall — APPROVED** [review]
